packages/shex-webapp/doc/shex-simple.js

```diff
@@ -50,10 +50,7 @@ var Getables = [
   {queryStringParm: "meta",         location: Caches.inputMeta.selection,   cache: Caches.inputMeta  },
 ];
 
-var QueryParams = Getables.concat([
-  {queryStringParm: "interface",    location: $("#interface"),       deflt: "human"     },
-  {queryStringParm: "regexpEngine", location: $("#regexpEngine"),    deflt: "threaded-val-nerr" },
-]);
+var QueryParams = Getables.concat([]);
```

Comment:
The `.concat()` is unnecessary now, this could also just be `var QueryParams = Getables;` as far as I can tell.